### PR TITLE
pull in a smaller gravatar icon

### LINKED ISSUE #1711: Should use smaller version of user icons in main view
On the front page, we currently get github and gravatar images using their native sizes, which are rather large (80x80 in the case of gravatar, up to 500x500 in the case of github)

We get the URL here:

https://github.com/iodide-project/iodide/blob/b45ecdc126a2db79cba30f3cd9eb5ceba2e1b6c4/server/views.py#L21

(the part of the query where we fetch the avatar URL is `owner__avatar`)

And render it in react here:

https://github.com/iodide-project/iodide/blob/b45ecdc126a2db79cba30f3cd9eb5ceba2e1b6c4/src/server/components/user-name.jsx#L24

This results in a large amount of unnecessary bandwidth being used and the pages loading more slowly.

For gravatar, we should be able to add `s=30` to the URL to get them at the right size: https://en.gravatar.com/site/implement/images/

Github seems to support the same, e.g. https://avatars3.githubusercontent.com/u/20569?v=4&s=30

To do this properly, I'd recommend a small helper function on the client that takes a URL and adds either a `?s=30` or a `&s=30` to it. I'm a bit torn on whether to do this on the client or server side, but am leaning towards doing it on the client. @robhudson @rafrombrc this would be another good bug to tackle.

### PR DESCRIPTION
**Pull request by teonbrooks**
this reduces the bandwidth of pulling in a gravatar.
closes #1711

### Pull Request checklist

- [x] **Documentation**: If this feature has or requires documentation, the relevant docs have been updated.
- [x] **Changelog**: This PR updates the [changelog](../CHANGELOG.md) with any user-visible changes.
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,6 +64,7 @@ typings/
 
 # Python cache directory
 .cache
+*.pyc
```
Comment (wlach) [inline .gitignore]: This should go in a separate PR, though I am not sure if it is needed? Not seeing any .pyc files in my iodide checkout

File: src/server/components/user-name.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const UserNameContainer = styled("a")`
   }
 `;
 
-const Avatar = ({ src }) => <img src={src} alt={src} />;
+const Avatar = ({ src }) => <img src={`${src}&s=30`} alt={src} />;
```
Comment (wlach) [inline src/server/components/user-name.jsx]: It might make sense to define the `30` in `src/shared/theme` as something like `THEME.defaultUserIconSize`. @hamilton might have opinions/suggestions on this.

**Comment by teonbrooks** [comment]
no test was included since this was a styling modification. let me know if you would like me to add a test

**Comment by teonbrooks** [comment]
@wlach, is this in line with what you were thinking?

**Comment by wlach** [comment]
Looks good! Testing locally, it seems like using a size of 40 gives better results (30 looks a little blurred), so let's go with that.
